quickstarts/kubernetes/nginx/config.yml

```diff
@@ -36,16 +37,20 @@ keywords:
   - config
 
 installPlans:
-  - third-party-nginx-ingress
+  - setup-prometheus
+  - setup-prometheus-agent-mode
```

Comment:
`setup-prometheus-agent-mode` does not appear to be an install plan. Could this be a typo?

Reply:
pending in https://github.com/newrelic/newrelic-quickstarts/pull/1503